megamek/src/megamek/common/Board.java

```diff
@@ -477,7 +476,7 @@ private void initializeHex(int x, int y, boolean event) {
         }
         
         // Internally handled terrain (inclines, cliff-bottoms)
-        initializeAutomaticTerrain(x, y);
+        initializeAutomaticTerrain(x, y, /* useInclines: */ true);
```

Comment:
Why include a comment here? Decreases readability IMO, especially when using an IDE that shows that automatically.

Reply:
It allows reading this without mouse-over which is better than having to mouse it. I like it when a parameter is otherwise completely meaningless. This call is so short that I can't see any issue with readability.

Reply:
I guess I've just gotten used to having it constantly showing when there's a constant used.